LambdaEngine/Source/Game/ECS/Systems/Rendering/RenderSystem.cpp

```diff
@@ -1134,6 +1131,27 @@ namespace LambdaEngine
 			m_TLASDirty = true;
 		}
 
+		// Remove extension
+		{
+			// Fetch the current instance and its extension index.
+			Instance& currentInstance = rasterInstances[instanceIndex];
```

Comment:
Can be a `const` reference. Same with `instanceKey` below.

Reply:
👍